src/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImpl.cs

```diff
@@ -2579,7 +2574,7 @@ private bool IsResolverNull
         {
             get
             {
-                return _xmlResolver == null || (LocalAppContextSwitches.ProhibitDefaultUrlResolver && !_xmlResolverIsSet);
+                return _xmlResolver == null || !_xmlResolverIsSet;
```

Comment:
Yes, when we initially ported Xml, we didn't have Configuration APIs, so we just set the variable to default value.